SpatialGDK/Source/SpatialGDK/Public/Utils/ErrorCodeRemapping.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Engine/EngineBaseTypes.h"
+#include <WorkerSDK/improbable/c_schema.h>
+
+namespace ENetworkFailure
+{
+	static inline ENetworkFailure::Type FromDisconnectOpStatusCode(uint8_t StatusCode)
+	{
+		// For full status code descriptions, see WorkerSDK\improbable\c_worker.h
+		switch (StatusCode)
+		{
+		case Worker_ConnectionStatusCode::WORKER_CONNECTION_STATUS_CODE_TIMEOUT:
```

Comment:
In the rest of the GDK, we don't indent switch statements, so might as well be consistent within the GDK.

Reply:
That is upsetting. Also not consistent with our style guide.

Reply:
I'll create a ticket for going through and fixing each of these. Or otherwise we could just update our style guide 🤷‍♂️? For now, I'll leave as is for consistency

Reply:
Raised https://improbableio.atlassian.net/browse/UNR-1232